ui/partials/perf/graphsctrl.html

```diff
@@ -54,6 +54,7 @@
   <div id="graph-tooltip" ng-class="{locked: selectedDataPoint}">
     <div class="body">
       <div>
+        <span id='close-popup'>X</span>
```

Comment:
We can reuse bootstrap's close button here, I believe. See this example:

https://github.com/mozilla/treeherder/blob/master/ui/partials/perf/testdatachooser.html#L2

(it looks a little nicer and doesn't require a custom CSS class)